src/components/UiText/index.js

```diff
@@ -70,15 +70,18 @@ const UiText = styled('p').attrs(({ variant }) => ({
   font-weight: ${styles.fontWeight};
   font-size: ${styles.fontSize};
   line-height: ${styles.lineHeight};
+  margin: ${({ margin }) => margin};
+  color: ${({ color }) => color};
 `;
 
 UiText.propTypes = {
   className: PropTypes.string,
+  margin: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
I find this so weird 😞

Reply:
Since props are spread, maybe there is no need for declaring every "styling propTypes"?

Reply:
I don't like it either, but it always bites me in the ass because one tends to pass `margin={0}` a lot.

If we don't declare it as a prototype then an outside product wouldn't know what props it accepts

Reply:
Dunno what's best here, because it also means that if we want to add another prop like this, we will have to write another proptype and release a new version, every time 🤔

Reply:
Yeah, but that makes sense to me, we'd do the same if we added a normal prop.

We're not covering all uses since we have `css`, only the ones that are so common they're worth having as a prop.

Reply:
I guess that makes sense yes
@tamarasaurus @keanan wdyt?

Reply:
@OctaveStoufflet since you're the only one that seems to check his Github notifications 😄

Reply:
As far as I understood, if we tend to pass `margin={0}` it make sense to pass it as a props, since it's more "practical" ? But... if we "always" wanna have margin = 0, why didn't we write directly
`margin : 0` in the styled rules ?

Reply:
well not "always" maybe.. but if we tend to have `margin: 0` we can choose to have `margin: 0` as a styled by default. and, if we wanna change this rules wile we are working on something, we can change the rules in the `styles.jsx` of the Component we are working on, no ?

Reply:
side note : thank you @ugogo i'm very honored to see that my insights matter.

Reply:
You're right :) we don't want `margin=0` to be the default, but it's common enough to be a prop on its own.

The question is mostly wether we should list all `style` props in `propTypes`, which I voted for

Reply:
Oh no @OctaveStoufflet sorry 😭 ❤️

Reply:
oooooh ok didn't understood the question sorry 
Yes we should indeed, to be aware